docs/diagrams/LessonClassDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
```

Comment:
Interestingly enough I think I drew this exact same class diagram in my PR 🤣 Where is this used? I drew because it's included in Model

Reply:
Oh I included this class diagram in the implementation details of the `add lesson` feature 😂 Maybe we can just use your diagram after we merge so that there will be no duplicates!